src/main/resources/db/migration/V20221104041400__create_table_projects.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE projects
+(
+    project_id NUMBER(6) PRIMARY KEY,
+    project_description VARCHAR(1000) NOT NULL ,
+    project_investment NUMBER(10, -3) NOT NULL ,
+    project_revenue NUMBER(10, -3),
+    CONSTRAINT prjct_description_min CHECK ( LENGTH(project_description) > 10 ),
```

Comment:
if is single constraint, it can be moved in field declaration

Reply:
but it's ok

Reply:
Ok, understood!